www/js/diary/infinite_scroll_list.js

```diff
@@ -101,7 +149,7 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
     $ionicLoading.show({
         template: $translate.instant('service.reading-server')
     });
-    Timeline.readAllConfirmedTrips(currEnd, ONE_WEEK).then((ctList) => {
+    Timeline.readAllConfirmedTrips(currEnd, ONE_DAY).then((ctList) => {
```

Comment:
Did we decide to read one day at a time? I thought this was just an experiment to check the loading response time...

Reply:
I will change it back now.